pkg/kev/converter/kubernetes/transform.go

```diff
@@ -205,8 +205,8 @@ func (k *Kubernetes) Transform() ([]runtime.Object, error) {
 	}
 
 	if renderedNetworkPolicy != nil {
-		stepNetworkPolicies := sg.Add("Creating networking policies")
-		stepNetworkPolicies.Success("Created networking policies")
+		stepNetworkPolicies := sg.Add("Creating network")
```

Comment:
As this is just a header for all things network related perhaps we can just call it `Networking`? Or to keep it in line with the above: `Converted network info`...